conda/environments/cudf_dev_cuda10.2.yml

```diff
@@ -16,12 +16,14 @@ dependencies:
   - numba>=0.49,!=0.51.0
   - numpy
   - pandas>=1.0,<1.2.0dev0
+  - pandavro
```

Comment:
since this `pandavro` dependency doesn't already in the [rapids-build-env](https://github.com/rapidsai/integration/blob/branch-0.16/conda/recipes/rapids-build-env/meta.yaml) package, a PR will need to be opened in the [integration repo](https://github.com/rapidsai/integration) before this one can be merged.

Reply:
I wasn't sure whether to add this dependency or not as this benchmark will not be run in CI.

Reply:
oh, good point. let me confer with @dillon-cullinan on that since he's heading the benchmark efforts.

Reply:
Should be fine to not include it. If you would like to update these benchmarks in the future to be ran with the other benchmarks (like libcudf), then we'll need to add it to one of the meta environments.